platform-stack/register_mqtt_sensor.sh

```diff
@@ -0,0 +1,351 @@
+#Provision a service group:EVChargingStation
```

Comment:
it's multiple service groups, not just ev charging

Reply:
added the service groups